rpa-home/modules/ROOT/pages/automation-tutorial-composer.adoc

```diff
@@ -0,0 +1,67 @@
+= Step 8: Integrate the Automation with Composer Flows
+
+After successfully publishing the process automation, you can connect your MuleSoft RPA account with MuleSoft Composer and trigger the automated process from a Composer flow. During this step you create a new MuleSoft Composer flow, configure a trigger to start the execution, connect your Composer account with your MuleSoft RPA account, and trigger the execution of an RPA Process. 
```

Comment:
```suggestion
After successfully publishing the process automation, you can connect your MuleSoft RPA account with MuleSoft Composer and trigger the automated process from a Composer flow. During this step, you create a new MuleSoft Composer flow, configure a trigger to start the execution, connect your Composer account with your MuleSoft RPA account, and trigger the execution of an RPA Process. 
```

Reply:
You use trigger as both a noun and a verb here. Should the first trigger be "start"?